pkg/operator/controllers/metrics.go

```diff
@@ -40,6 +40,14 @@ var (
 		},
 		[]string{"BuildName"},
 	)
+	GameServersUnhealthyCounter = registry.NewCounterVec(
```

Comment:
Could we add this to the table at the bottom of the monitoring section?
https://github.com/PlayFab/thundernetes/blob/main/docs/howtos/monitoring.md

Reply:
tracked in #280